test/integration/api/OrderAPI.spec.ts

```diff
@@ -1139,21 +1135,21 @@ describe('OrderAPI', () => {
       let nonComponentToken: StandardTokenMockContract;
 
       beforeEach(async () => {
-        nonComponentToken = await deployTokenAsync(provider, issuanceOrderMaker);
+        nonComponentToken = await deployTokenAsync(web3.currentProvider, issuanceOrderMaker);
         zeroExOrder.makerAssetData = SetUtils.encodeAddressAsAssetData(nonComponentToken.address);
       });
 
       test('throws', async () => {
         return expect(subject()).to.be.rejectedWith(
-          `Token address at ${nonComponentToken.address} is not a component ` +
+          `Token address at ${nonComponentToken.address.toLowerCase()} is not a component ` +
```

Comment:
What were you saying about the checksum here?

Reply:
On any lowercase address you need to compare you need to call web3.utils.toChecksum on it to convert to mix case